docs/_adapters/cilium.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Meshery Adapter for Cilium
+name: Meshery Adapter for Cilium
+mesh_name: Ciluim
+earliest_version: v1.10.6
+port: 10012/tcp
+project_status: beta
+adapter_version: v0.0.0
+lab: cilium-meshery-adapter
+github_link: https://github.com/meshery/meshery-cilium
+image: /assets/img/service-meshes/cilium.svg
+permalink: service-meshes/adapters/cilium
+---
+
+{% include adapter-status.html %}
+
+### Features
+
+1. {{page.mesh_name}} Lifecycle Management
+1. Workload Lifecycle Management
+   1. Using Service Mesh Standards
+      1. Service Mesh Performance (SMP)
+         1. Prometheus and Grafana connections
+      1. Service Mesh Interface (SMI)
+1. Configuration Analysis, Patterns, and Best Practices
+   1. Custom Service Mesh Configuration
+
+## Lifecycle management
+
+The {{page.name}} can install **{{page.version}}** of {{page.mesh_name}}. A number of sample applications can be installed using the {{page.name}}.
+
+### Install {{ page.mesh_name }}
```

Comment:
we don't need the Install for now.

Reply:
You could workload management here with a couple of sample apps like we have on the Tanzu adapter page.

Reply:
We might want to crack open another issue to track this item.

You're right that we do want to be clear about the state of the adapter.